### PR TITLE
Port core-functions from wiki

### PR DESCRIPTION
**Pull request by PromoFaux**
Signed-off-by: Adam Warner <adamw@rner.email>

(It must be October)

### CODE REVIEW

**Comment by Mcat12** [comment]
I think this can be split out into a new "Core" or "CLI" section, as "Getting Started" focuses just on installing and updating.

**Comment by PromoFaux** [comment]
Fair shout, so as a new subscriber heading?

**Comment by Mcat12** [comment]
Ya, at the same level as FTLDNS and Guides.

File: mkdocs.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@ pages:
   - 'Installation': 'main/basic-install.md'
   - 'Post-Install': 'main/post-install.md'
   - 'Updating': 'main/update.md'
+- 'Pi-hole Core':
+  - '`pihole` command': 'main/pihole-command.md'
```
Comment (Mcat12) [inline mkdocs.yml]: The new path is `core/pihole-command.md`

**Comment by Mcat12** [comment]
@PromoFaux I fixed a few extra things, so I'll wait until you or someone else checks it over.

**Pull request merged**